test/core/test_varexpr.py

```diff
@@ -78,14 +90,67 @@ def assert_syntax_error(ctx: VarExprContext, expr: str):
             ctx.evaluate(expr)
 
     # noinspection PyMethodMayBeStatic
-    def test_invalid_expressions(self):
+    def test_that_harmless_builtins_are_callable(self):
+        ctx = VarExprContext(dataset)
+
+        result = ctx.evaluate("min(A, B)")
+        self.assertIsInstance(result, xr.DataArray)
+
+        result = ctx.evaluate("max(A, B)")
+        self.assertIsInstance(result, xr.DataArray)
+
+        result = ctx.evaluate("ceil(B)")
+        self.assertIsInstance(result, xr.DataArray)
+
+    # noinspection PyMethodMayBeStatic
+    def test_that_evil_builtins_not_callable(self):
+        ctx = VarExprContext(dataset)
+
+        with pytest.raises(
+            VarExprError,
+            match="name 'open' is not defined",
+        ):
+            ctx.evaluate("open('mycode.py', 'w')")
+
+        with pytest.raises(
+            VarExprError,
+            match="name 'input' is not defined",
+        ):
+            ctx.evaluate("input()")
+
+        with pytest.raises(
+            VarExprError,
+            match="name '__module__' is not defined",
+        ):
+            ctx.evaluate("__module__")
+
+        with pytest.raises(
+            VarExprError,
+            match="name '__import__' is not defined",
+        ):
+            ctx.evaluate("__import__('evilmodule')")
+
+    # noinspection PyMethodMayBeStatic
+    def test_invalid_expression_cases(self):
         ctx = VarExprContext(dataset)
 
         with pytest.raises(
             VarExprError,
-            match="'DataArray' object has no attribute 'dims'",
+            match="name 'my_secret_var' is not defined",
+        ):
+            ctx.evaluate("my_secret_var")
+
+        with pytest.raises(
+            VarExprError,
+            match="'DataArray' object has no attribute 'values'",
+        ):
+            ctx.evaluate("A.values")
```

Comment:
Do you want to know why I disabled it or how the disabling works?

- why I disabled it: the `xr.DataArray.values` property loads all (!) data of a variable and returns it as a numpy array (this can easily cause oom errors)
- how it works: the wrapper `ExprVar` does not have such an attribute